### PR TITLE
Addition of script to make tokens tradable

### PR DESCRIPTION
**Pull request by juniset**
Simple JS script to call the `TokenPriceRegistry` and set a list of tokens as `tradable`.

### CODE REVIEW

File: scripts/set_tradable_tokens.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  const { deployer } = manager;
+  const deploymentWallet = deployer.signer;
+  const { config } = configurator;
+
+  const TokenPriceStorageWrapper = await deployer.wrapDeployedContract(TokenPriceRegistry, config.modules.TokenPriceRegistry);
+
+  const tokens = JSON.parse(fs.readFileSync(input, "utf8"));
+  const tradable = Array(tokens.length).fill(true);
+
+  // const res = await TokenPriceStorageWrapper.getTradableForTokenList(tokens);
```
Comment (elenadimitrova) [inline scripts/set_tradable_tokens.js:29]: We should check the `tradable` flag is not set on any of the submitted tokens as the contract reverts when a manager disables a tradable token.

File: scripts/set_tradable_tokens.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  // const res = await TokenPriceStorageWrapper.getPriceForTokenList(tokens);
```
Comment (elenadimitrova) [inline scripts/set_tradable_tokens.js:30]: We should also check there is a price set for the token we set the flag for as the contracts don't check existence of token in list before setting `tradable` flag on it meaning we can end up with tokens with 0 prices that are tradable.

**Review by elenadimitrova — CHANGES_REQUESTED** [review]

**Comment by gergold** [comment]
I've addressed all your comments except the last one (we can end up with tokens with 0 prices that are tradable), I'm checking with Tom and Adam what will be the process, set tradable or price first.

**Comment by elenadimitrova** [comment]
Thanks @gergold can you also fix build pls `npm run lint:js` is failing

**Review by elenadimitrova — APPROVED** [review]

**Pull request merged**